dateutil/test/test_parser.py

```diff
@@ -125,11 +127,23 @@ def testDateCommandFormatWithLong(self):
                                    tzinfos={"BRST": long(-10800)}),
                              datetime(2003, 9, 25, 10, 36, 28,
                                       tzinfo=self.brsttz))
+
     def testDateCommandFormatIgnoreTz(self):
         self.assertEqual(parse("Thu Sep 25 10:36:28 BRST 2003",
                                ignoretz=True),
                          datetime(2003, 9, 25, 10, 36, 28))
 
+    def testDateCommandFormatRespectTz(self):
```

Comment:
Pretty sure this will always fail on Windows, so this would need to be changed in the following ways:

1. Add a `@skipIf` to skip this test on Windows.
2. Use [`dateutil.tests._common.TZEnvContext`](https://github.com/dateutil/dateutil/blob/master/dateutil/test/_common.py#L159) for this, rather than just setting the `os.environ` directly like this.